pkg/translator/prometheus/normalize_name.go

```diff
@@ -197,6 +197,10 @@ func CleanUpString(s string) string {
 	return strings.Join(strings.FieldsFunc(s, func(r rune) bool { return !unicode.IsLetter(r) && !unicode.IsDigit(r) }), "_")
 }
 
+func CleanUpStringForMetricName(s string) string {
```

Comment:
Please rename to `RemovePromForbiddenRunes()` and add a comment to describe the difference with *CleanUpString()*